application/repositories.go

```diff
@@ -1,11 +1,27 @@
-package remoteworkitem
+package application
 
 import (
 	"github.com/almighty/almighty-core/app"
 	"github.com/almighty/almighty-core/criteria"
 	"golang.org/x/net/context"
 )
 
+// WorkItemRepository encapsulates storage & retrieval of work items
+type WorkItemRepository interface {
+	Load(ctx context.Context, ID string) (*app.WorkItem, error)
+	Save(ctx context.Context, wi app.WorkItem) (*app.WorkItem, error)
+	Delete(ctx context.Context, ID string) error
+	Create(ctx context.Context, typeID string, fields map[string]interface{}) (*app.WorkItem, error)
+	List(ctx context.Context, criteria criteria.Expression, start *int, length *int) ([]*app.WorkItem, error)
+}
+
+// WorkItemTypeRepository encapsulates storage & retrieval of work item types
+type WorkItemTypeRepository interface {
+	Load(ctx context.Context, name string) (*app.WorkItemType, error)
+	Create(ctx context.Context, extendedTypeID *string, name string, fields map[string]app.FieldDefinition) (*app.WorkItemType, error)
+	List(ctx context.Context, start *int, length *int) ([]*app.WorkItemType, error)
+}
+
 // TrackerRepository encapsulate storage & retrieval of tracker configuration
 type TrackerRepository interface {
```

Comment:
By defining the `TrackerRepository` interface here, the packages are getting more coupled, right?